src/extend-config/index.ts

```diff
@@ -15,13 +15,13 @@ const extendConfig = (json: FabloConfigJson): FabloConfigExtended => {
     chaincodes: chaincodesJson,
   } = json;
 
-  const capabilities = configTransformers.getNetworkCapabilities(networkSettingsJson.fabricVersion);
+  const networkSettings = configTransformers.transformNetworkSettings(networkSettingsJson);
+  const capabilities = configTransformers.getNetworkCapabilities(networkSettings.fabricVersion);
   const rootOrg = configTransformers.transformRootOrgConfig(rootOrgJson);
-  const orgs = configTransformers.transformOrgConfigs(orgsJson);
+  const orgs = configTransformers.transformOrgConfigs(orgsJson, networkSettings);
```

Comment:
hm... why org needs whole networkSettings block? Can't we just pass `fabloRest` ?